client/my-sites/migrate/step-import-or-migrate.jsx

```diff
@@ -47,28 +45,38 @@ class StepImportOrMigrate extends Component {
 	};
 
 	getJetpackOrUpgradeMessage = () => {
-		const { sourceSite, sourceHasJetpack, isTargetSiteAtomic } = this.props;
+		const { sourceSite, sourceHasJetpack, isTargetSiteAtomic, translate } = this.props;
 
 		if ( ! sourceHasJetpack ) {
 			const sourceSiteDomain = get( sourceSite, 'domain' );
 			return (
 				<p>
-					You need to have{ ' ' }
-					<a href={ `https://wordpress.com/jetpack/connect/install?url=${ sourceSiteDomain }` }>
-						Jetpack
-					</a>{ ' ' }
-					installed on your site to be able to import everything.
+					{ translate(
+						"You need to have <a href='%(jetpackInstallLink)s'>Jetpack</a> installed on your site to be able to import everything.",
```

Comment:
I think the best practice here is to pass the [anchor tag as a component](https://github.com/Automattic/i18n-calypso#mixing-strings-and-markup).

Reply:
Thanks! Updated! Also thanks for sharing this tip!

Reply:
We're still mixing in markup here...